spec/requests/providers/home_address/non_uk_home_addresses_controller_spec.rb

```diff
@@ -0,0 +1,138 @@
+require "rails_helper"
+
+RSpec.describe Providers::HomeAddress::NonUkHomeAddressesController, :vcr do
+  let(:legal_aid_application) { create(:legal_aid_application, :with_applicant) }
+  let(:applicant) { legal_aid_application.applicant }
+  let(:provider) { legal_aid_application.provider }
+  let(:address) { applicant.address }
```

Comment:
@skdejong similar to above, this should be applicant.home_address however it won't work until ap-4833 is merged.